### ISSUE TITLE
Cannot connect to Windows 10 after Creators Update (1703)

### ISSUE THREAD

**Issue by osos** [FreeRDP/Remmina#1186]
* LinuxMint 18.1
* PPA-version of remmina 
* Cinnamon/gnome desktop
* Connecting to Windows 10 Pro 1703
* Connecting via RDP

I have been using Remmina to connect to Windows 10 Pro 1607 since month with no problems. After upgrading with the Creators Update (1703) for Windows 10 Pro I am unable to connect.

Others seem to have issues, see: https://social.technet.microsoft.com/Forums/office/en-US/e23a8b8d-f84d-4bb5-aad6-211d83a9aa89/windows-10-1703-breaks-remoteapp-remote-desktop-gateway?forum=winserverTS

**Comment by osos** [comment]
Just found this: https://www.solaris-cookbook.eu/linux/remmina-windows-10-fixed-unable-to-connect-to-rdp-server/

It states: 

> Somewhere, somehow the key must have changed.
> 
> I fixed that by deleting the freerdp known_hosts entry.
> 
` vi ~/.freerdp/known_hosts`
> 
> Remove the host entry for the particular server.
> 
> When trying to reconnect it will ask you to re-authorize the host.

Will try once I return home, and report here.

**Comment by osos** [comment]
Unfortunately it didn't work :-(

Please advise

**Comment by giox069** [comment]
I'm using Remmina to connect to Windows 10 1703 but I have no problems.
Are you using a remote desktop Gateway as described on the 1st link you posted?
Try to execute remmina from a terminal window: open the terminal window, type remmina and press enter. Connect to your PC. After the connection fails, please return to the terminal windows and copy all the text you see and paste here. Thank you.

**Comment by osos** [comment]
Turns out that the upgrade disabled the firewall-rules allowing Windows to receive RDP requests and thus effectively disabling remote desktop, despite otherwise enabled!

**Comment by alancarwile** [comment]
I started having issues with Remmina on ubuntu 14.04 after **windows update (pushed by IT) put 1703** on my Windows 10 system.  I tried many of the things mentioned with no luck.  I never changed the actual windows system remote desktop config since I could access it from a separate windows system and from a mac, but not from ubuntu.  I had **remmina from the PPA** but no luck - not enough.  Finally I noticed I was **not running the latest from the PPA**, so I updated remmina and other "recommended" changes, (but staying still on Ubuntu 14.04) and the **new remmina works fine** to remote into that system.  The **working version is 1.2.0-rcgit-27** (git rcgit-27), whereas the **“*26*” version failed** with symptoms mentioned earlier.  View version with "remmina -v" or "remmina -v" (one lowercase, one uppercase, V).
Whew! Working again!
Alan
